Contents/mods/AwayFromZomboid/media/lua/client/AwayFromZomboid/AwayFromZomboid.lua

```diff
@@ -353,4 +358,14 @@ end
 
 -- Init hook
 
-Events.OnConnected.Add(AwayFromZomboid.init)
\ No newline at end of file
+Events.OnCreatePlayer.Add(AwayFromZomboid.init)
```

Comment:
Just to confirm, this is also ran when a player already exists, right? Not only during initial character creation?

Reply:
Exactly